### PR TITLE
typo

### PR DESCRIPTION
**Pull request by CJ-Wright**
What is a `sample`?

### CODE REVIEW

**Comment by hhslepicka** [comment]
A ```sample``` is just a field! And in the magic universe of MongoDB it can be anything you want! :+1: 
From a simple string to a reference to a sample UID registered in ```amostra```.

File: doc/source/searching.rst [hunk 1]
```diff
@@ -259,7 +259,7 @@ Here are just a few examples:
 =========================================== ============================================================
 syntax                                                          meaning
 =========================================== ============================================================
-``db(sample={'$exists': True})``            headers that include a custom metadata field labeled 'color'
```
Comment (tacaswell) [inline doc/source/searching.rst]: can you fix the typo the other way (making both 'sample')?

**Comment by tacaswell** [comment]
@CJ-Wright can I push commits to your branch?

**Comment by CJ-Wright** [comment]
Go nuts @tacaswell 
@hhslepicka I was kidding, since the name of the field was `sample` but it referenced color.

**Comment by tacaswell** [comment]
Thanks @CJ-Wright !

**Comment by CJ-Wright** [comment]
Of course!

**Pull request merged**